doc/plugins/org.polarsys.capella.ui.doc/html/Installation Guide/How to install Capella and Addons.mediawiki

```diff
@@ -113,11 +117,16 @@ Check that the extraction process has finished without error.
 *  Agree and click on Finish
 *  Restart when prompted
 
-'''Verification''' 
+'''For a shared installation, we recommend to do the following:'''
+* The administrator install Capella in a read-only directory
+* The configurator will create a shared master configuration and populate it as needed (add/remove/update addons)

```

Comment:
its a bit confusing between 'context' and 'configuration'. what is a shared master configuration, a context as described in dropins?

maybe move section describing context/configuration from dropin section to make it work on both case?

Reply:
Hi ali, you can switch to other topic.
we will continue this after some other tests. (see minh tu mail)